WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/ContactPreview.kt

```diff
@@ -19,20 +19,25 @@ import android.graphics.BitmapFactory
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.remember
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
 import androidx.wear.tiles.ActionBuilders
 import androidx.wear.tiles.ModifiersBuilders
 import androidx.wear.tiles.ResourceBuilders
 import com.example.wear.tiles.messaging.MessagingRepo
 import com.example.wear.tiles.messaging.MessagingTileRenderer
 import com.example.wear.tiles.messaging.bitmapToImageResource
-import com.example.wear.tiles.tools.IconSizePreview
-import com.example.wear.tiles.tools.LayoutPreview
+import com.google.android.horologist.compose.tools.LayoutPreview
 
 val emptyClickable = ModifiersBuilders.Clickable.Builder()
     .setOnClick(ActionBuilders.LoadAction.Builder().build())
     .setId("")
     .build()
 
+@Preview(
+    backgroundColor = 0xff000000, showBackground = true, widthDp = 100, heightDp = 100
+)
+annotation class IconSizePreview
```

Comment:
Should this and :

```
import com.example.wear.tiles.tools.WearPreviewDevices
import com.example.wear.tiles.tools.WearPreviewFontSizes
import com.example.wear.tiles.tools.WearPreviewLocales
```

exist in Horologist?

Reply:
Ideally yes, but https://issuetracker.google.com/issues/233511976